include/openssl/stack.h

```diff
@@ -545,7 +565,7 @@ namespace internal {
 template <typename Stack>
 struct DeleterImpl<
     Stack, typename std::enable_if<StackTraits<Stack>::kIsConst>::type> {
-  static void Free(Stack *sk) { sk_free(reinterpret_cast<_STACK *>(sk)); }
+  static void Free(Stack *sk) { OPENSSL_sk_free(reinterpret_cast<_STACK *>(sk)); }
```

Comment:
Should this be:
```suggestion
  static void Free(Stack *sk) { OPENSSL_sk_free(reinterpret_cast<OPENSSL_STACK *>(sk)); }
```
?